### PR TITLE
Add Mobile Developer job opening

### PR DESCRIPTION
**Pull request by maryiailiusina**
We begin looking for mobile developers

### CODE REVIEW

File: jobs/mobile-developer.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- For the development process we have Kanban desks, communication in channels and calls, including direct communication with clients. Tools that help us are
+Slack, Trello, Notion. We also us Holacracy framework for organizational governance
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit
+
+## How a good candidate looks
+
+- You have solid experience in anything we use - Kotlin, Swift or React Native
+- You want to work with React Native, even if you have not much experience with it
+- ***(? tech stuff- like more details about frameworks and technologies we would like to see in background ?)***
```
Comment (maryili) [inline jobs/mobile-developer.md:24]: We should add here specific technologies and details

File: jobs/mobile-developer.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- Your English skills allow you to feel comfortable communicating in it (verbally and written) the whole working day
+- You know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication
+- Learning process is an important part of your professional activities
+- Teamwork is not an abstract word for you, but a set of principles and agreements you follow
+- You value the quality of the code
```
Comment (maryili) [inline jobs/mobile-developer.md:29]: I think we should formulate it better, but not sure I can make it properly. Any ideas?

File: jobs/mobile-developer.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+We are looking for experienced and initiative Mobile Developers to grow our team and develop meaningful products together. We value openness and 
+knowledge sharing, believe that communication is a key and develop the remote culture for around 6 years. We would like to work with someone who shares our values,
+acts independently and proactively.
+
+
+## How the job looks
+
+- We have jobs both for Android and iOS developers, experienced (or ready to improve their skills) in React Native. We work with Kotlin + RxJava + Dagger for Android
+and Swift + RxSwift for iOS. We also use React Native and going to use it more
```
Comment (vshkl) [inline jobs/mobile-developer.md:11]: The tech stack is inaccurate. Let's better say this way:
>We primarily use Kotlin for Android and Swift for iOS, yet certain Java or Objective-C knowledge might be useful.

The reason for that is that `RxJava`, `RxSwift`, and `Dagger` are quite random technologies and they don't really cover represent the stack we are (or may be) using for our native projects. We should either broaden the list (which is not what we need here I suppose) or stop with just used languages. I've added a part about `Java` and `Objective-C` experience due to the fact that, for instance, the current native project has a good part of the code written in those, the previous small one we used to work with had been written in `Java` and `Objective-C` completely. Also, all the native insides of the `react-native` project are written in `Java` and `Objective-C` still.
TL;DR: the ability to at least read and somehow maintain code in `Java` and `Objective-C` might be if not required but a useful skill to have.

**Review by vshkl — COMMENTED** [review]
@maryili, I've left several comments on tech-related stuff in the PR. I haven't covered all the items yet. I'll allocate some additional time later this week.
JFYI, all the formulations I've written are just suggestions. You can rewrite them however you like of course. 🙂

File: jobs/mobile-developer.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+We are looking for experienced and initiative Mobile Developers to grow our team and develop meaningful products together. We value openness and knowledge sharing, believe that communication is a key and develop the remote culture for around 6 years. We would like to work with someone who shares our values, acts independently and proactively.
+
+
+## How the job looks
+
+- We have jobs both for Android and iOS developers, experienced (or interested to learn) in React Native. We primarily use Kotlin for Android and Swift for iOS, yet certain Java or Objective-C knowledge might be useful.
+- You will work with a team of developers (3-5 people) on the current project developing new features.
+- Usually our projects last for about 1,5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time.
+- Salary 𝚫: 24k—48k USD.
```
Comment (dzhlobo) [inline jobs/mobile-developer.md:11]: Should we move it higher up?

File: jobs/mobile-developer.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- For development we usually use Kanban or similar process in Trello, Pivotal of Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use [Holacracy](https://en.wikipedia.org/wiki/Holacracy) for organizational governance.
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities.
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## How a good candidate looks
+
+- You have solid experience either in Android or iOS.
+- You wish to work with React Native, even if you have not much experience in it.
+- You have an opinion on how to architect the app and can compare MVP with MVVM.
+- You know how different components of mobile OSes work: push notifications, keystore & keychain, background tasks, GPS api, etc.
```
Comment (dzhlobo) [inline jobs/mobile-developer.md:21]: @vshkl, I've added some of the OS APIs but maybe you have better opinion on what we should list here based on what APIs you've been working recently?

File: jobs/mobile-developer.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- Salary 𝚫: 24k—48k USD.
+- For development we usually use Kanban or similar process in Trello, Pivotal of Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use [Holacracy](https://en.wikipedia.org/wiki/Holacracy) for organizational governance.
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities.
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## How a good candidate looks
+
+- You have solid experience either in Android or iOS.
+- You wish to work with React Native, even if you have not much experience in it.
+- You have an opinion on how to architect the app and can compare MVP with MVVM.
```
Comment (dzhlobo) [inline jobs/mobile-developer.md:20]: @vshkl, is it true that people very often use Rx* when they implement MVVM architecture?

File: jobs/mobile-developer.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+- Teamwork is not an abstract word for you, but a set of principles and agreements you follow.
+
+
+## What you'll definitely need to work with us
+
+- Experience in writing native mobile apps for Android or iOS.
+- Experience in working with React Native or desire to learn it.
+- English knowledge to communicate with colleagues and clients.
+- Communication skills to collaborate remotely.
+- Independence and initiative - we won't control how you work, but we expect the work to be done.
```
Comment (dzhlobo) [inline jobs/mobile-developer.md:37]: @maryili, we maybe create a false representation that people in the team work independently but in fact I encourage collaboration with teammates, pair- and mob-programming, and technical discussions.

File: jobs/mobile-developer.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+- 50% compensation for self-development activities: courses, conferences, books, educational platforms, English, sports.
+- Remote work and flexible schedule (actually no schedule at all - you decide when you work).
+- 32 paid days off per year - you can use them for vacations, occasional days off, and public holidays, only workdays count.
+- Full coverage of sick leaves.
+- 200 USD per year for medical expenses.
+
+## How to apply 💌
+
+In order to apply send us a message to [join@datarockets.com](mailto:join@datarockets.com) with your CV. In the message tell us about you and your experience, share some examples of your code.
```
Comment (dzhlobo) [inline jobs/mobile-developer.md:57]: @vshkl, @maryili, I like to ask candidates some questions they answer when they apply. Usually that helps me to understand a level of developer and provokes a lot of discussions on an interview. E.g. if a candidate tells me that they use MVVM architecture for code organization w/o detail I may ask many additional questions: where does a business logic go, how does communication with API look like and where code responsible for this lie, how do you make sure API-communication code is independent from UI code, etc.

What do you think if we ask some questions in advance? Questions I may think of:
- How would you organize you code so when a business logic changes we don't have to rewrite code related to UI? Can you share an example from your practice?
- How do you write tests: which practices do you use and which you avoid? What kind of tests do you write and why?

@vshkl, do you have better suggestion for questions?

**Comment by dzhlobo** [comment]
I've changed styling a little bit (mostly added periods at the end of sentences 🤓) and added some engineering requirements. I think we might add a couple of questions for prospective candidates and we're good to go with this text.

File: jobs/mobile-developer.md (hunk 6 above)
Comment (vshkl) [reply jobs/mobile-developer.md:20]: @dzhlobo, kind of. However, on Android, there is an _official_ approach for that: `LiveData`.

File: jobs/mobile-developer.md (hunk 5 above)
Comment (vshkl) [reply jobs/mobile-developer.md:21]: Those are fine. I'd add `db`, probably, but it doesn't matter much, 'cuz we can't list all of them anyway.

File: jobs/mobile-developer.md (hunk 6 above)
Comment (dzhlobo) [reply jobs/mobile-developer.md:20]: Got it. Let's leave it as is then and we'll discuss it on an interview.

File: jobs/mobile-developer.md (hunk 4 above)
Comment (maryiailiusina) [reply jobs/mobile-developer.md:11]: we decided to leave it here

File: jobs/mobile-developer.md (hunk 5 above)
Comment (dzhlobo) [reply jobs/mobile-developer.md]: Good point 👍. Added.

File: jobs/mobile-developer.md (hunk 8 above)
Comment (maryiailiusina) [reply jobs/mobile-developer.md]: great idea, will add this!
